### PR TITLE
V1.2 UI Applicant List and Interview List 

### LINKED ISSUE #45: As a Hiring Manager, I want to be able to view all interviews, so that I can have quick access to the information

### LINKED ISSUE #46: As a Hiring Manager, I want to be able to view all applicants, so that I can have quick access to their information

### PR DESCRIPTION
**Pull request by Chen1x**
closes #45
closes #46

### CODE REVIEW

File: src/main/java/seedu/address/logic/Logic.java [hunk 1]
```diff
@@ -33,6 +34,10 @@ public interface Logic {
     /** Returns an unmodifiable view of the filtered list of persons */
     ObservableList<Person> getFilteredPersonList();
 
+    /** Returns an unmodifiable view of the filtered list of interviews */
+    ObservableList<Interview> getFilteredInterviewList();
```
Comment (kiwibang) [inline src/main/java/seedu/address/logic/Logic.java:38]: I like the informative name 👍

File: src/main/java/seedu/address/ui/InterviewCard.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package seedu.address.ui;
```
Comment (kiwibang) [inline src/main/java/seedu/address/ui/InterviewCard.java:1]: Very cleanly implemented class. Props for implementing the UI components of our app 👍

File: src/main/java/seedu/address/ui/InterviewListPanel.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Interview} using a {@code InterviewCard}.
+     */
+    class InterviewListViewCell extends ListCell<Interview> {
+        @Override
+        protected void updateItem(Interview interview, boolean empty) {
+            super.updateItem(interview, empty);
+
```
Comment (kiwibang) [inline src/main/java/seedu/address/ui/InterviewListPanel.java:39]: Might want to add some test cases using the [Equivalence Partition method](https://nus-cs2103-ay2324s1.github.io/website/schedule/week10/topics.html#w10-4-test-cases-equivalence-partitioning).

File: src/main/java/seedu/address/ui/InterviewListPanel.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            if (empty || interview == null) {
+                setGraphic(null);
+                setText(null);
```
Comment (kiwibang) [inline src/main/java/seedu/address/ui/InterviewListPanel.java:42]: Same for these too! Might want to add some meaningful test cases here!

**Review by kiwibang — APPROVED** [review]
Overall LGTM. Very clean and easy to follow code. Small issue might be the need to test the important methods of your classes more in order to catch bugs early on. Other than that, thanks for the great work 😸

**Review by Jonyxzx — APPROVED** [review]
LGTM!

**Pull request merged**